Installation/Windows/Templates/NSIS.template.in

```diff
@@ -650,6 +650,8 @@ FunctionEnd
 
 Function InstallOptionsPage2
    IfSilent 0 continueUI
+   ; this would be called inside the UI code:
+   Call stop_old_service

```

Comment:
I have no idea if this change is related to the other changes in this PR, and if it is required.

Reply:
Should it be removed then?